internal/dao/tests/pubkey_resource_test.go

```diff
@@ -33,7 +33,7 @@ func createPubkeyResourceAzure() *models.PubkeyResource {
 	}
 }
 
-func setupResource(t *testing.T) (dao.PubkeyResourceDao, context.Context) {
+func setupPubkeyResource(t *testing.T) (dao.PubkeyResourceDao, context.Context) {
```

Comment:
I've also renamed this, as when I searched for usage, this got me confused and I thought I'm renaming wrong method.
Hope it's ok :)